src/main/scala/com/fulcrumgenomics/cmdline/ClpGroups.scala

```diff
@@ -20,17 +20,16 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
- *
  */
 
 package com.fulcrumgenomics.cmdline
 
 import dagr.sopt.cmdline.ClpGroup
 
-
```

Comment:
Would be nice to wrap these declarations in an `object ClpGroups` and access them as `ClpGroups.Personal` to give them a clearer namespace.